src/styles/overrides/_se-events.scss

```diff
@@ -0,0 +1,163 @@
+.SECalendarViewletDetails {
+  ul {
+    list-style-type: none;
+    @extend .dg_card;
+    text-align: left;
+
+    li {
+      .eventcalevent {
+        @extend h2;
+      }
+
+      .eventcaldate,
+      .eventcaldate:before,
+      .eventcaltime:before {
+        margin-right: $margin-small;
+      }
+
+      .eventcaldate:before {
+        @include icon-content($icon-calendar);
+      }
+
+      .eventcaltime:before {
+        @include icon-content($icon-clock);
+      }
+    }
+  }
+}
+
+.dg_actions-list {
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+
+    li {
+      display: inline-block;
+      margin-right: $default-margin;
+      margin-bottom: $default-margin;
+
+      a {
+        @extend .dg_button;
+        display: inline-block;
+
+        &:before {
+          margin-right: $margin-smallest;
+        }
+      }
+
+      &.action a {
+        background-image: none;
+      }
+
+      &.download a {
+        &:before {
+          @include icon-content($icon-download);
+        }
+      }
+
+      &.reminder a {
+        &:before {
+          @include icon-content($icon-reminder);
+        }
+      }
+      &.share a {
+        &:before {
+          @include icon-content($icon-share);
+        }
+      }
+
+      &.email a {
+        &:before {
+          @include icon-content($icon-email);
+        }
+      }
+
+      &.register a {
+        &:before {
+          @include icon-content($icon-register);
+        }
+      }
+      &.twitter a {
+        &:before {
+          font-family: "Font Awesome 5 Brands";
+          font-weight: 900;
+          content: $icon-twitter;
+        }
+      }
```

Comment:
font-weight needs to be $font-boldest also line 91

Reply:
looks like there is similar repeated code that could be combined like mentioned above.